src/provider/httpDefinitionProvider.ts

```diff
@@ -0,0 +1,101 @@
+import * as vscode from 'vscode';
+import { DocumentStore } from '../documentStore';
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+// thanks to https://github.com/abierbaum/vscode-file-peek
+
+export class HttpDefinitionProvider implements vscode.DefinitionProvider {
+  private isDebug: boolean;
+  protected fileSearchExtensions: string[] = ['json'];
+
+  constructor(readonly documentStore: DocumentStore) {
+    this.isDebug = false;
+  }
+
+  provideDefinition(
+    document: vscode.TextDocument,
+    position: vscode.Position
+  ): vscode.ProviderResult<vscode.Definition | vscode.LocationLink[]> {
+    const working_dir = path.dirname(document.fileName);
+    const word = document.getText(document.getWordRangeAtPosition(position));
+    const line = document.lineAt(position);
+
+    this.log_debug('====== peek-file definition lookup ===========', '');
```

Comment:
There would be too many log_debug outputs that replace a simple breakpoint. Are you having problems with the setup for debugging? Can I help?

Reply:
I just personal preference, I like logs more :). I moved to breakpoints at this moment.